acceptance/builder_test.go

```diff
@@ -0,0 +1,184 @@
+package acceptance
+
+import (
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/lifecycle/api"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+)
+
+var (
+	builderDockerContext = filepath.Join("testdata", "builder")
+	builderBinaryDir     = filepath.Join("testdata", "builder", "container", "cnb", "lifecycle")
+	builderImage         = "lifecycle/acceptance/builder"
+	builderUserID        = "1234"
+)
+
+func TestBuilder(t *testing.T) {
+	h.SkipIf(t, runtime.GOOS == "windows", "builder acceptance tests are not yet supported on Windows")
```

Comment:
A bit of context to why Windows tests aren't supported for some phases - for phases that require access to a registry, enabling the test container (where the builder binary is running) to connect with the test registry container was challenging because of networking, but we found a way around this (see analyzer acceptance tests). Also the build phase doesn't need to connect to a registry, so this concern might not be an issue. 

However given that it's a bit complicated to get things working cross-OS, it might make sense to start with Linux tests for now and make them compatible with Windows later.

Reply:
Yeah, i think these tests should work fine other OS too. We are using docker for test environment and like you mentioned this phase no need any specific feature depends on OS. I will Add a FIXME comment for not forget later.

Reply:
I think it's worth adding this FIXME comment 😊

Reply:
added.

Reply:
Let's try taking out this Skip and see what happens :)

Reply:
(I'm expecting, if there are failures, that we might need to wrap some strings in `ctrPath`)